staking_deposit/cli/exit_transaction_mnemonic.py

```diff
@@ -0,0 +1,115 @@
+
+
+@click.command(
+    help=load_text(['arg_exit_transaction_mnemonic', 'help'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda x: closest_match(x, list(ALL_CHAINS.keys())),
+        choice_prompt_func(
+            lambda: load_text(['arg_exit_transaction_mnemonic_chain', 'prompt'], func=FUNC_NAME),
+            list(ALL_CHAINS.keys())
+        ),
+    ),
+    default=MAINNET,
+    help=lambda: load_text(['arg_exit_transaction_mnemonic_chain', 'help'], func=FUNC_NAME),
+    param_decls='--chain',
+    prompt=choice_prompt_func(
+        lambda: load_text(['arg_exit_transaction_mnemonic_chain', 'prompt'], func=FUNC_NAME),
+        # Since `prater` is alias of `goerli`, do not show `prater` in the prompt message.
+        list(key for key in ALL_CHAINS.keys() if key != PRATER)
+    ),
+)
+@load_mnemonic_arguments_decorator
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda num: validate_int_range(num, 0, 2**32),
+        lambda: load_text(['arg_exit_transaction_mnemonic_start_index', 'prompt'], func=FUNC_NAME),
+    ),
+    default=0,
+    help=lambda: load_text(['arg_exit_transaction_mnemonic_start_index', 'help'], func=FUNC_NAME),
+    param_decls="--validator_start_index",
+    prompt=lambda: load_text(['arg_exit_transaction_mnemonic_start_index', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda validator_indices: validate_validator_indices(validator_indices),
+        lambda: load_text(['arg_exit_transaction_mnemonic_indices', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_exit_transaction_mnemonic_indices', 'help'], func=FUNC_NAME),
+    param_decls='--validator_indices',
+    prompt=lambda: load_text(['arg_exit_transaction_mnemonic_indices', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    default=0,
+    help=lambda: load_text(['arg_exit_transaction_mnemonic_epoch', 'help'], func=FUNC_NAME),
+    param_decls='--epoch',
+)
+@jit_option(
+    default=os.getcwd(),
+    help=lambda: load_text(['arg_exit_transaction_mnemonic_output_folder', 'help'], func=FUNC_NAME),
+    param_decls='--output_folder',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+)
+@click.pass_context
+def exit_transaction_mnemonic(
+        ctx: click.Context,
+        chain: str,
+        mnemonic: str,
+        mnemonic_password: str,
+        validator_start_index: int,
+        validator_indices: Sequence[int],
+        epoch: int,
+        output_folder: str,
+        **kwargs: Any) -> None:
+
+    chain_settings = get_chain_setting(chain)
+    num_keys = len(validator_indices)
+    key_indices = range(validator_start_index, validator_start_index + num_keys)
+
+    click.echo(load_text(['msg_creation_start']))
+    # We assume that the list of validator indices are in order and increment the start index
+    for key_index, validator_index in zip(key_indices, validator_indices):
+        credential = Credential(
```

Comment:
This can also occur when using the existing-mnemonic flow. 

There are a number of locations where a ValidationError is thrown that will result in the stack trace and error message.

I think wrapped try/catches might be a bit much and perhaps we should remove the ValidationError from more expected locations such as an invalid mnemonic. 

Though if we do want to go that route, we might want to do that in a separate pull request.

What do you think?

Reply:
I agree. I created #18 to keep track of this.